pkg/api/live_handler.go

```diff
@@ -0,0 +1,58 @@
+package api
+
+import (
+	"strings"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/live"
+	"github.com/grafana/grafana/pkg/util"
+)
+
+// LiveHTTPHandler will process any requets to /api/live/*
+func (hs *HTTPServer) LiveHTTPHandler(c *models.ReqContext) {
```

Comment:
I don't see how this handler gets exercised from the frontend, I'd need to know how it's supposed to be used before I can approve.